db/src/main/kotlin/edu/artic/db/Playable.kt

```diff
@@ -1,9 +1,35 @@
 package edu.artic.db
 
+import android.net.Uri
```

Comment:
using an android `Uri` in a viewmodel means we might have trouble testing it without robolectric tests.

Reply:
We benefit from strong typing in Uri that a `String` typealias couldn't offer. I'll change it to `okhttp3.HttpUrl`, as proposed offline.

Reply:
No, wait, we can just switch back to Strings. Should've thought of that earlier....